content/docs/getting-started/installation/kubernetes/powerstore/prerequisite/_index.md

```diff
@@ -16,6 +16,7 @@ The following requirements must be met before installing the CSI Driver for Powe
 - Insecure registries are defined in Docker or other container runtime for CSI drivers that are hosted in a non-secure location.
 - Ensure that your nodes support mounting NFS volumes if using NFS.
 - For NVMe support the preferred multipath solution is NVMe native multipathing. The [Dell Host Connectivity Guide](https://elabnavigator.dell.com/vault/pdf/Linux.pdf) describes the details of each configuration option.
+- If using Host Based NFS - Install necessary nfs packages and ensure nfs-server and nfs-mountd services are active and running on all nodes.
```

Comment:
Is nfs-mountd spelled correctly?

Reply:
It is, "nfs-mountd" is the name of the service.